### ISSUE TITLE
Exposure days list always return one element.

### ISSUE THREAD

**Issue by albertovecina** [DP-3T/dp3t-sdk-android#193]
If I'm not mistaken, the exposure days list is always returning a one element list with the last exposure day:

https://github.com/DP-3T/dp3t-sdk-android/blob/739bf165b0d47c8ccdea6f3dbbd2252cb521aef4/dp3t-sdk/sdk/src/main/java/org/dpppt/android/sdk/internal/storage/ExposureDayStorage.java#L91

Am I wrong? Is this a bug? How can I get the full exposure days list?

Thanks

**Comment by pantic79** [comment]
I think it would be useful to return all the exposures and that this kind of filtering is performed by the apps using the SDK.

**Comment by simonroesch** [comment]
You are correct, it makes more sense to do such a filtering on the app side. This will be changed with version 1.1 of the SDK: https://github.com/DP-3T/dp3t-sdk-android/pull/204
